catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/RecyclerViewFragment.java

```diff
@@ -436,6 +452,12 @@ public void onPositiveButtonClick(DialogInterface dialog, String textInput) {
 				.show();
 	}
 
+	protected void showMergeAlert(List<T> selectedItems) {
+		//CATROID-156 showing Toast will be implemented here
```

Comment:
please no unnecessary todo comments. im sure youll be able to locate the method again for 156 without this here.